frontend/dockerfile/cmd/dockerfile-frontend/Dockerfile

```diff
@@ -20,7 +20,7 @@ ARG TARGETPLATFORM
 ENV TARGETPLATFORM=$TARGETPLATFORM
 RUN --mount=target=. --mount=type=cache,target=/root/.cache \
   --mount=source=/tmp/.ldflags,target=/tmp/.ldflags,from=version \
-  CGO_ENABLED=0 go build -o /dockerfile-frontend -ldflags "-d $(cat /tmp/.ldflags)" -tags "$BUILDTAGS netgo static_build osusergo" ./frontend/dockerfile/cmd/dockerfile-frontend && \
+  CGO_ENABLED=0 GO111MODULE=on go build -mod vendor -o /dockerfile-frontend -ldflags "-d $(cat /tmp/.ldflags)" -tags "$BUILDTAGS netgo static_build osusergo" ./frontend/dockerfile/cmd/dockerfile-frontend && \
```

Comment:
Will this change in `go1.12` ? If the dockerfiles don't use `go mod` anyway atm we don't need `-mod vendor`. We could just add comments with a reminder that we need to enable it when `go mod` is enabled.